src/flamenco/runtime/program/fd_bpf_loader_v3_program.c

```diff
@@ -362,6 +372,28 @@ execute( fd_exec_instr_ctx_t * instr_ctx, fd_sbpf_validated_program_t * prog ) {
     .alloc               = { .offset = 0UL }
   };
 
+  ulong trace_sz = 4 * 1024 * 1024;
+  fd_vm_trace_entry_t * trace = NULL;
+  fd_vm_trace_context_t trace_ctx;
+  (void) trace_sz;
+  (void) trace;
+  (void) trace_ctx;
+
+#ifdef FD_DEBUG_SBPF_TRACES
```

Comment:
(note for later)

In belt-sanding we can make this a run-time argument along with the signature, so we don't have to recompile